.github/workflows/coverage.yml

```diff
@@ -0,0 +1,47 @@
+name: Code coverage
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install codecov
+      shell: bash
+      run: |
+          echo "::set-env name=CC::gcc-9"
+          echo "::set-env name=CXX::g++-9"
+          python -u -m pip install git+https://github.com/nmoinvaz/codecov-python.git@master
+
+
+    - name: Create Build Environment
+      shell: bash
+      run: cmake -E make_directory ${{runner.workspace}}/build
+
+    - name: Configure
+      shell: bash
+      working-directory: ${{runner.workspace}}/build
+      run: cmake $GITHUB_WORKSPACE -DCMAKE_BUILD_TYPE=Debug -DPISA_BUILD_TOOLS=OFF -DENABLE_COVERAGE=ON ..
```

Comment:
What's `GITHUB_WORKSPACE` here?

Reply:
I think this should point to the root git directory, so the same as `..`. I think I used this at some point, and then changed the working directory and used `..` instead. So I think this is unnecessary. I'm actually a bit surprised it doesn't just fail, because it gets two paths but maybe it just ignores one of them.

`GITHUB_WORKSPACE` seems to be the same as `${{runner.workspace}}` but it's set by the `checkout` action. Not sure if there's a case in which it can be different.